class.jetpack.php

```diff
@@ -4082,7 +4082,7 @@ function admin_page_load() {
 			$this->error = __( 'Cheatin&#8217; uh?', 'jetpack' );
 			break;
 		case 'access_denied' :
-			$this->error = __( 'You need to authorize the Jetpack connection between your site and WordPress.com to enable the awesome features.', 'jetpack' );
+			$this->error = __( 'Would you mind telling us why you did not complete the Jetpack connection in this <a href="http://surveys.jetpack.me/cancelled-connection">1 question survey</a>?', 'jetpack' ) . '<br><small>' . __( 'A Jetpack connection is required for our free Security, and Traffic features to work.', 'jetpack' ) . '</small>';
```

Comment:
Doesn't it feel to you like those two sentences are disconnected? Perhaps we should join them into one paragraph and rephrase? HTML inside of the strings is totally fine, you can even add the `br` and `small` wrappers in, but please move the link out and use `sprintf`.